modules/tools/default.nix

```diff
@@ -0,0 +1,15 @@
+{ soxin, config, lib, ... }:
+
+with lib;
+{
+  options = {
+    soxin.toolsModules = mkOption {
```

Comment:
```suggestion
    soxin.tools = mkOption {
```

Reply:
@rissson I had named this `toolsModule` because it conflicted with tools in `list.nix`. I remember you mentioned a workaround in our last meeting, can you please remind me what it was ?

Reply:
We said we would import the `settings` module as a whole, and then be able to import `tools` and `programmingLanguages` alongside it.

Reply:
This is done